Dockerfile.armhf

```diff
@@ -131,10 +131,10 @@ RUN ln -sv $PWD/contrib/completion/bash/docker /etc/bash_completion.d/docker
 # Get useful and necessary Hub images so we can "docker load" locally instead of pulling
 COPY contrib/download-frozen-image-v2.sh /go/src/github.com/docker/docker/contrib/
 RUN ./contrib/download-frozen-image-v2.sh /docker-frozen-images \
-	armhf/buildpack-deps:jessie@sha256:eb2dad77ef53e88d94c3c83862d315c806ea1ca49b6e74f4db362381365ce489 \
-	armhf/busybox:latest@sha256:016a1e149d2acc2a3789a160dfa60ce870794eea27ad5e96f7a101970e5e1689 \
-	armhf/debian:jessie@sha256:ac59fa18b28d0ef751eabb5ba4c4b5a9063f99398bae2f70495aa8ed6139b577 \
-	armhf/hello-world:latest@sha256:9701edc932223a66e49dd6c894a11db8c2cf4eccd1414f1ec105a623bf16b426
+	buildpack-deps:jessie@sha256:dd86dced7c9cd2a724e779730f0a53f93b7ef42228d4344b25ce9a42a1486251 \
+	busybox:latest@sha256:bbc3a03235220b170ba48a157dd097dd1379299370e1ed99ce976df0355d24f0 \
+	debian:jessie@sha256:287a20c5f73087ab406e6b364833e3fb7b3ae63ca0eb3486555dc27ed32c6e60 \
+	hello-world:latest@sha256:be0cd392e45be79ffeffa6b05338b98ebb16c87b255f48e297ec7f98e123905c
```

Comment:
This is the one case where `download-frozen-image-v2.sh` won't be smart enough as-is to use the correct image (just like Docker ATM isn't smart enough as-is).  See https://github.com/moby/moby/issues/34875 for more details (and `download-frozen-image-v2.sh` now uses a very similar algorithm to Docker's own for choosing).

Reply:
Right! Things will be a little bit complicated when we talking about arm arch, since it acrosses wider ecosystem from embedded, mobile/IoT to the enterprise. For the topic itself, clearly we need a `variant` to location the exact arm image such as `armhf` and `armel`. Current problem is with `GOARCH` we only tell `arm64` or `arm`, we've realized this so that's why we have a PR https://github.com/containerd/containerd/pull/1575 merged into `containerd` to handle this, that PR uses `/proc/cpuinfo` to get `v5 v6 v7 v8` info of arm. So IMO, probably we need to add the `variant` support in `download-frozen-image-v2.sh` in terms of `arm 32bit` case. currently I prefer to let it to be still existing one , and enhance that after all the multi-arch futures for Arm arch are ready. For me, I don't care about the 32-bit at all, my github account is `arm64b` meaning 'it's arm, but only 64b(it)` 🎃. But if you have any suggestions here, I am happy to take account of it.